lib/data/external/AwsClient.js

```diff
@@ -43,7 +43,10 @@ class AwsClient {
                     const log = createLogger(reqUids);
                     log.error('err from data backend',
                     { error: err, dataStoreName: this._dataStoreName });
```

Comment:
And it may be a good time to create a logging helper since there are a lot of similar such calls in this file.